### PR TITLE
update documentation + xml changes

### PR DESCRIPTION
**Pull request by rvanheest**
~~fixes EASY-~~

#### When applied it will
* update the documentation according to the former changes
* xml changes that I learned from @jo-pol 

@DANS-KNAW/easy for review

### CODE REVIEW

File: src/main/scala/nl/knaw/dans/easy/multideposit/package.scala [hunk 1]
```diff
@@ -161,17 +161,15 @@ package object multideposit {
...
      * @param encoding the encoding applied to this xml
      */
     @throws[IOException]("in case of an I/O error")
     def writeXml(elem: Elem, encoding: Charset = encoding): Unit = {
-      val header = s"""<?xml version="1.0" encoding="$encoding"?>\n"""
-      val data = new PrettyPrinter(160, 2).format(elem)
-
-      file.write(header + data, encoding)
+      file.getParentFile.mkdirs()
+      XML.save(file.toString, elem, encoding.toString, xmlDecl = true)
```
Comment (janvanmansum) [inline src/main/scala/nl/knaw/dans/easy/multideposit/package.scala:172]: Neat! 👍

File: src/test/scala/nl/knaw/dans/easy/multideposit/BlackBoxSpec.scala [hunk 2]
```diff
@@ -125,4 +128,13 @@ class BlackBoxSpec extends UnitSpec with BeforeAndAfter with MockFactory {
         )
     }
   }
+
+  def transformer(label: String) = new RuleTransformer(new RewriteRule {
```
Comment (janvanmansum) [inline src/test/scala/nl/knaw/dans/easy/multideposit/BlackBoxSpec.scala]: Does it need to be called `transformer`? That doesn't convey what it does. It seems to be deleting all the elements named `"$label"`. Is that correct? If so, maybe `removeElementsByName` or something, would better fit the bill.

**Review by janvanmansum — APPROVED** [review]

File: src/main/scala/nl/knaw/dans/easy/multideposit/package.scala (hunk 1 above)
Comment (rvanheest) [reply src/main/scala/nl/knaw/dans/easy/multideposit/package.scala:172]: thank you @jo-pol!

File: src/test/scala/nl/knaw/dans/easy/multideposit/BlackBoxSpec.scala (hunk 2 above)
Comment (rvanheest) [reply src/test/scala/nl/knaw/dans/easy/multideposit/BlackBoxSpec.scala]: good one! fixed it

**Pull request merged**